.github/workflows/klayout.yml

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'klayout'
+
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: '0 0 * * 5'
+  workflow_dispatch:
+  repository_dispatch:
+    types: [ klayout ]
+
+env:
+  DOCKER_BUILDKIT: 1
+
+jobs:
+
+  klayout:
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - run: echo "$(pwd)/.github/bin" >> $GITHUB_PATH
+
+    - run: dockerBuild pkg:klayout klayout pkg
+    - run: dockerBuild klayout     klayout
+
+    - run: dockerTestPkg klayout
+    # klayout test currently fails due to lack on X environment
+    # - run: dockerTest klayout
```

Comment:
Is having a DISPLAY (X server) a requirement even for `--version` or `--help`? If that is the case, I believe it should be reported upstream. Meanwhile, it's ok to have this commented about, as long as we test/use the image periodically.

Reply:
@umarcor 
Will consider it for the coming days.
But shouldn't it be good also to have a test environment with dummy X framebuffer included ?
I did try to look at gtkwave for how to handle it for klayout, but that does not seem to have a smoke test.

Reply:
Yes. We can create a dummy environment with a dummy X framebuffer. I believe we can use https://github.com/mviereck/x11docker/ for that. However, I don't know what can we do after starting the container with an X server. That will allow us to do a screenshot, at most. Do you have any idea about how to smoke test these graphical applications? In the case of gtkwave, we can smoke test some of the waveform conversion tools, but I'm unsure about gtkwave itself.